src/components/TextField/TextField.tsx

```diff
@@ -515,10 +516,10 @@ export function TextField({
   }
 
   function handleClick({target}: React.MouseEvent) {
-    if (containsAffix(target)) {
+    if (containsAffix(target) || isInputFocused) {
```

Comment:
I thought this would be coming from state but it's actually a utility. I'm switching to `focus` from state and that should fix the broken test